doc/whats_new/v1.5.rst

```diff
@@ -284,6 +284,10 @@ Changelog
 - |API| :func:`metrics.brier_score_loss` deprecated the keyword argument `y_prob`
   in favor of `y_proba`. `y_prob` will be removed in version 1.7.
   :pr:`28092` by :user:`Adam Li <adam2392>`.
+ 
+- |Enhancement| :func:`sklearn.metrics.mean_tweedie_deviance` now supports Array API compatible
+  inputs.
+  :pr:`28106` by :user:`Thomas Li <lithomas1>`
```

Comment:
Instead of putting this entry in the section about `sklearn.metrics`, let's move it to the top level section named "Support for Array API" in the same file.